includes/Products/class-fb-feed-generator.php

```diff
@@ -189,7 +200,7 @@ public function get_default_column_names() {
 	private function create_feed() {
 		$result = \WC_Facebookcommerce_Utils::$fbgraph->create_feed(
 			facebook_for_woocommerce()->get_integration()->get_product_catalog_id(),
-			array( 'name' => self::FEED_NAME )
+			array( 'name' => self::FEED_NAME, 'schedule' => $this->get_feed_upload_schedule() )
```

Comment:
I think that we will end up with `update_schedule` but that is something we need to figure out later.

Reply:
Based on [Product Catalog Product Feed](https://developers.facebook.com/docs/marketing-api/reference/product-catalog/product_feeds/) documentation:

- `schedule`:  this will be a full replace operation where items not present in the feed file will be removed, new items will be created and existing items will be updated.
- `update_schedule`: The uploads would only update the items in the feed or create new ones. No items would be deleted. This is useful for sending price and availability updates for selected items in the feed.

`schedule` should works for us unless we plan to create two separate feeds for each case @haszari @budzanowski

Reply:
Let's look at this next - when we have a reliable engine for generating the feed file :)